modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBAdminNavigationDisplayContext.java

```diff
@@ -129,6 +132,29 @@ public JSONArray getKBFolderDataJSONArray() throws PortalException {
 			));
 	}
 
+	public long getKBObjectToMoveParent() throws PortalException {
+		String kbObjectToMoveType = ParamUtil.getString(
+			_httpServletRequest, "kbObjectToMoveType");
```

Comment:
If we use `class.getSimpleName()` we should rename this to `kbObjectToMoveClassName`.

Reply:
It is not a classname, just the strings "article" and "folder" defined in constants. Do you still think we should rename it to `kbObjectToMoveClassName`?